CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# How to contribute
+
+## Open an issue
+
+Feel free to open an issue, No matter whether you found a bug, request a feature or have questions.
+
+1. Please check existing open and closed issues first and feel free to comment and re-open them.
+2. Use a meaningful title
+3. Provide enough context, a concrete use case and steps to reproduce bugs
+
+## Contributing code and documentation
+
+1. Create a fork of the repository (You need a GitHub account)
```

Comment:
We can link https://docs.github.com/de/get-started/quickstart/fork-a-repo and https://github.com/join here.

Reply:
I suggest that https://docs.github.com/en/get-started/quickstart/contributing-to-projects already contains the first suggestion.

I'll happily add the 2nd suggestion for joining GitHub.